### DISCUSSION TITLE
my calendar does not show my progress even if I commit.

### DISCUSSION THREAD

**Discussion by bgomiss** [community/community#41067] [New to GitHub]
it is like this since 2nd December. What can be the problem. I have two different branches and I m working on second branch lately

**Comment by airtower-luna** [comment]
> I have two different branches and I m working on second branch lately

That explains it: The contribution graph shows only commits on the default branch of a repository. They should show up once you merge into the default branch. :slightly_smiling_face: 

See: [Contributions that are counted](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#contributions-that-are-counted)

**Reply by bgomiss** [reply]
but I want my first main branch stay as it is the first version of my app. If I merge will I lose it?

**Reply by airtower-luna** [reply]
Merging would (slightly simplified) add the commits from the branch you're currently working to the default branch.

How you want to manage your branches is ultimately up to you, but the common convention is:

* The default branch (usually called `main`) is where current development happens, often by merging short-lived feature (or "topic") branches.
* Versions of a software are marked by tags.

If you're new to these concepts I recommend looking at the [Git Book](https://git-scm.com/book/en/v2). Chapters 2 and 3 explain the most important concepts (including branches, merging, and tags).

From what you've mentioned I think it'd make sense for you to tag the first version, e.g. `v1.0` (or whatever fits with the versioning you use), and then continue working on the default branch.

**Reply by bgomiss** [reply]
thank you very much
